admin/src/graphql/communityStatistics.js

```diff
@@ -4,12 +4,14 @@ export const communityStatistics = gql`
   query {
     communityStatistics {
       totalUsers
-      activeUsers
       deletedUsers
       totalGradidoCreated
       totalGradidoDecayed
-      totalGradidoAvailable
-      totalGradidoUnbookedDecayed
+      totalAvailable {
```

Comment:
```suggestion
      active {
```
naming is not self explanatory.

Reply:
I choose `dynamicStatisticsFields`